bucket/glimpse.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": "0.2.0",
+    "description": "Glimpse is an open source image editor based on the GNU Image Manipulation Program. The goal is to experiment with new ideas and expand the use of free software.",
+    "homepage": "https://glimpse-editor.github.io/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "PFiles\\Glimpse Image Editor\\Glimpse 0.2.0 (64-bit)",
+            "url": "https://github.com/glimpse-editor/Glimpse/releases/download/v0.2.0/glimpse-0.2.0-x64.msi",
+            "hash": "420dbbe11fef2a1d741083eb72f99732b9aec620d1d8274f9fb7a30e650030ff"
+        },
+        "32bit": {
+            "extract_dir": "PFiles\\Glimpse Image Editor\\Glimpse 0.2.0",
+            "url": "https://github.com/glimpse-editor/Glimpse/releases/download/v0.2.0/glimpse-0.2.0-i686.msi",
+            "hash": "2163403ca13d27be399b7fc1348719b2a9a4ddc4bc8c4b00aa8edd608bacfdbd"
+        }
+    },
+    "installer": {
+        "script": [
```

Comment:
Why is this needed in first place?

Reply:
This is a pruned variant of the GIMP's installer script, no longer dealing with files that aren't present. I'm not totally sure why theirs does what it does, and I did not test a no installer script variant.